### ISSUE TITLE
[MU4 Issue] Missing properties options for pedal element

### ISSUE THREAD

**Issue by cgpqci** [musescore/MuseScore#14382]
**Describe the bug(?)**
So, there are some things I want to point out and would like to suggest being added as a feature regarding with pedal lines. I'll be bundling them all here. To list:

![image](https://user-images.githubusercontent.com/90231404/200259683-bbf8c321-984b-4d27-b76f-dd67b61bb904.png)
1. Line visible option is no longer available in MU4 and instead replaced by Show pedal symbol option. It could've been very handy in replicating single pedal symbol in engraving without a distracting line (_like shown above_).
2. As an alternative to No. 1, An invisible line should be added as one of the options in Line type properties (![image](https://user-images.githubusercontent.com/90231404/200266212-d7eb12f8-0b66-441a-8195-0dbd7fda377e.png)) to replicate the same engraving.
3. Allow diagonal option is also no longer available too, but this feature doesn't work either way for the pedal in particular in MU3 (_see first pedal element_).  It would be nice if a passage like below is possible without resorting to a workaround like using text elements:

![image](https://user-images.githubusercontent.com/90231404/200263711-49d1f5c2-608f-4288-9af3-c48e975e5de3.png)
(_reproduced in MU4_)

That's only about it. I hope this is put into consideration. Thank you!

**Comment by LuisLiszt** [comment]
Agreed. Also there's no option to disable continuation text if the Pedal goes over multiple staffs...

**Comment by Tantacrul** [comment]
We're going to be redesigning the piano pedal widget (and inspector options) completely in an upcoming release. I'll make sure to address all these points.

**Comment by Jojo-Schmitz** [comment]
Some other lines are missing those settings too, not just pedal lines

**Comment by Jojo-Schmitz** [comment]
Came up again in https://musescore.org/en/node/385283

**Comment by Jojo-Schmitz** [comment]
And 3 3/4 years ago already in https://musescore.org/en/node/329628
